src/nimutils/ansi.nim

```diff
@@ -1,10 +1,10 @@
-import tables, options
+import tables, options, os
 
 
 var showColors      = true
 
 proc setShowColors*(val: bool) =
-  showColors = val
+  showColors = val and getEnv("NO_COLOR") == ""
```

Comment:
any value of `NO_COLOR` when set will disable colors. can change it to `existsEnv` but this seemed more graceful